packages/backend-modules/mail/templates/membership_owner_autopay_failed.html

```diff
@@ -93,8 +93,8 @@
                       Ihre Mitgliedschaft am {{grace_end_date}} zu deaktivieren.
                     </p>
                     <p>
-                      So weit muss es nicht kommen: Begleichen Sie den ausstehenden 
-                      Jahresbeitrag {{prolong_url}} , wenn Sie möchten, auch mit einer 
+                      So weit muss es nicht kommen: <a href="{{prolong_url}}">Begleichen Sie den ausstehenden 
+                      Jahresbeitrag</a>, wenn Sie möchten, auch mit einer 
```

Comment:
same here
```suggestion
                      So weit muss es nicht kommen: 
                      <a href="{{prolong_url}}">Begleichen Sie den ausstehenden 
                      Jahresbeitrag</a>
                      , wenn Sie möchten, auch mit einer 
```

Reply:
Won't this add a space between the link and the comma?